### DISCUSSION TITLE
PSA Car Controller + Docker + Synology

### DISCUSSION THREAD

**Discussion by nuiler** [flobz/psa_car_controller#761] [General]
I would like to get the "PSA CAR Controller" running again as a Docker under Synology.
Unfortunately, I can't access the web interface. I have already changed the port but this does not help.

Is there a step-by-step guide for this? What do I need to consider?

![image](https://github.com/flobz/psa_car_controller/assets/16724275/ca8b4461-8f65-4fad-89bb-d4668d9744d0)

![image](https://github.com/flobz/psa_car_controller/assets/16724275/289fc573-3bbf-478a-a56e-d27d98fda856)

flobz-psa_car_controller

date,stream,content
2024/02/15   11:12:10,stdout,"2024-02-15 10:12:10,778 :: INFO ::  * Running on http://0.0.0.0:6000/ (Press   CTRL+C to quit)

Can anyone possibly help me? Gladly also via Teamviewer.

**Comment by sod75** [comment]
I'm running it on a docker container on my synology, though with docker-compose. I haven't done anything specific .
The install is pretty straightforward. https://github.com/flobz/psa_car_controller/blob/master/docs/Docker.md and from what you posted that seems correct

There is an issue ongoing with the first time authentication due to stellantis making a change to their api. : https://github.com/flobz/psa_car_controller/issues/733#issuecomment-1939444099 with a workaround in progress

but that shouldn't stop you from getting to the web interface. 
Are you sure it's not a network/IP address  issue, e.g. is the contained on the network, ...
I use  just http://192.168.0.91:3000/ to access it from my local network.

**Comment by nuiler** [comment]
I can now access the web interface. I had to use port 4000.

How can I now use the workaround?

https://github.com/flobz/psa_car_controller/issues/733#issuecomment-1939555182
